BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -209,6 +213,80 @@ private void ConvertFuncStatic(Entity funcStatic)
 			funcStatic.ClassName = "func_brush";
 		}
 
+		private void ConvertFuncPlat(Entity entity)
+		{
+			var moveDistance = 0f;
+			var brushThickness = GetBrushThickness(entity);
+
+			if (float.TryParse(entity["height"], out var height))
+				moveDistance = height + brushThickness;
+			else if (float.TryParse(entity["lip"], out var lip))
+				moveDistance = -(lip - 2 - (brushThickness * 2));
+
+			if (string.IsNullOrEmpty(entity.Name))
+			{
+				entity.Name = $"plat{platIndex}";
+				CreatePlatTrigger(entity);
+				platIndex++;
+			}
+			entity.ClassName = "func_door";
+			entity["lip"] = moveDistance.ToString();
+			entity["movedir"] = "-90 0 0";
+			entity["spawnpos"] = "1";
+			entity["spawnflags"] = "0";
+			entity["wait"] = "-1";
+		}
+
+		private void CreatePlatTrigger(Entity entity)
+		{
+			var trigger = new Entity
+			{
+				ClassName = "trigger_multiple",
+				Model = entity.Model,
+				Spawnflags = 1,
+				Origin = new Vector3(entity.Origin.X, entity.Origin.Y, entity.Origin.Z + 2)
+			};
+			trigger["parentname"] = entity.Name;
+			sourceEntities.Add(trigger);
+
+			AddPlatTriggerConnections(entity, trigger);
+		}
+
+		private void AddPlatTriggerConnections(Entity plat, Entity trigger)
+		{
+			var connection = new Entity.EntityConnection()
+			{
+				name = "onStartTouch",
+				target = plat.Name,
+				action = "close",
+				param = null,
+				delay = 0,
+				fireOnce = -1
+			};
+			trigger.connections.Add(connection);
+
+			var connection2 = new Entity.EntityConnection()
+			{
+				name = "onFullyClosed",
+				target = plat.Name,
+				action = "open",
+				param = null,
+				delay = 3,
```

Comment:
The platform never resets in q3 as long as a player is touching it. It resets 1 second after you step off. I can _probably_ recreate this with entity logic in the future but it would need a bunch more entity connections that will look pretty messy. This is just a placeholder value to make it work for the meantime. I can comment it as such.